### PR TITLE
VMware Fusion translated to markdown - google codein

### PR DESCRIPTION
**Pull request by ikkeboe**

### CODE REVIEW

File: content/guides/virtualizing/VMware Fusion.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file.
+
+Under Virtual Machine Library, which you can use the shortcut **Shift**+**cmd**+**L**, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](/files/new-vm-step-1.png)
+![](/files/new-vm-step-2.png)
+![](/files/new-vm-step-3.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/VMware Fusion.md]: Where are these images?

File: content/guides/virtualizing/VMware Fusion.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+**TODO**
+
+##### Step 6. Specifying Disk space
+
+**TODO**
+
+##### Step 7. Adjusting Memory.
+
+**TODO**
```
Comment (waddlesplash) [inline content/guides/virtualizing/VMware Fusion.md]: Could you possibly fill in all these TODOs?

File: content/guides/virtualizing/VMware Fusion.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+Close the dialog box and select the partition you just created. Ensure that the partition is correct.
+
+##### Step 13. Installing Haiku.
+
+Begin the installation! Reboot once you are done, you should be able to see the Haiku desktop after you reboot. If you are stuck at the installation screen after reboot, disconnect the iso from the virtual disk drive and try again.
+
+##### Step 14. You are now finished.
+
+ you have successfully installed Haiku! Have fun and refer to the Haiku User Guide if you encounter any problems.
```
Comment (waddlesplash) [inline content/guides/virtualizing/VMware Fusion.md]: Grammar nit: no space at beginning of line, capital `y`.

File: content/guides/virtualizing/VMware Fusion.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+The required files can be found at on the Get-Haiku page of this website which is located [here](http://www.haiku-os.org/get-haiku). Both the vmdk and iso are available there, do select the closest mirror to enjoy higher transfer rates. Verify using the checksums to make sure that the downloaded files are not corrupted as they are big files. VMware Fusion is available for free on their website which can be found [here](http://www.vmware.com/products/fusion/).
+
+###### Go to section: 
+
+* [Running Haiku from a VM image.](#user-content-running-haiku-from-a-vm-image)
+* [Installing and running Haiku from an ISO image.](#user-content-installing-and-running-haiku-from-an-iso-image)
+* [Additional Steps.](#user-content-additional-steps)
+* [Troubleshooting.](#user-content-troubleshooting)
+
+
```
Comment (waddlesplash) [inline content/guides/virtualizing/VMware Fusion.md]: Remove double-newline.

File: content/guides/virtualizing/vmware-fusion.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+##### Step 1. Downloading Haiku.
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file.
+
+Under Virtual Machine Library, which you can use the shortcut **Shift**+**cmd**+**L**, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](/files/new-vm-step-1.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: You still haven't added these files to the repository.

File: content/guides/virtualizing/vmware-fusion.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file.
+
+Under Virtual Machine Library, which you can use the shortcut **Shift**+**cmd**+**L**, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](https://www.haiku-os.org/files/new-vm-step-1.png)
+![](https://www.haiku-os.org/files/new-vm-step-2.png)
+![](https://www.haiku-os.org/files/new-vm-step-3.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: No, sorry, as I said before, you must add the images to the repository in the proper directory.

File: content/guides/virtualizing/vmware-fusion.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
++++ type = "article" title = "VMWare Fusion" date = "2016-12-26T14:27:10.000Z" tags = [] +++
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: It must be on separate lines, just as all the other files are.

File: content/guides/virtualizing/vmware-fusion.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file.
+
+Under Virtual Machine Library, which you can use the shortcut **Shift**+**cmd**+**L**, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](../../images/new-vm-step-1.png)
+![](../../images/new-vm-step-2.png)
+![](../../images/new-vm-step-3.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: The files must go in the /static/images directory. See a similar commit which does this: https://github.com/haiku/website/commit/9d48282a1e7f71c5a62f63accc4a19e33f3fabad

Comment (richienyhus) [inline content/guides/virtualizing/vmware-fusion.md]: Should the "new-vm-step-3.png" image be located further down at step 4 ?

Comment (ikkeboe) [inline content/guides/virtualizing/vmware-fusion.md]: Yes, it makes more sense that way.

File: content/guides/virtualizing/vmware-fusion.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+##### Step 1. Downloading Haiku.
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file.
+
+Under Virtual Machine Library, which you can use the shortcut **Shift**+**cmd**+**L**, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](../../../static/images/new-vm-step-1.png)
+![](../../../static/images/new-vm-step-2.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: No, there is no `static` in the final path, an example of a proper path would be `/files/images/new-vm-step-1.png`. I linked you to a correct example; why is this so hard?

Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: @ikkeboe Also, why did you move the images? They were just fine in the files directory; might as well keep them there instead of moving them.

File: content/guides/virtualizing/vmware-fusion.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+##### Step 1. Downloading Haiku.
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file.
+
+Under Virtual Machine Library, which you can use the shortcut **Shift**+**cmd**+**L**, click Add and then New.
+
+##### Step 2. Creating a virtual machine.
+
+![](/images/new-vm-step-1.png)
+![](/images/new-vm-step-2.png)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-fusion.md]: @ikkeboe why does it make more sense that way? Right now, the images directory mostly contains site assets, while the `files` directory contains all the content images. Why move them and make the repository larger?

Comment (ikkeboe) [inline content/guides/virtualizing/vmware-fusion.md]: sorry I have made a bad commit
